packages/generators/remove-generator.ts

```diff
@@ -32,13 +32,15 @@ export default class RemoveGenerator extends Generator {
 				webpackOptions: {}
 			}
 		};
-
-		let configPath = path.resolve(process.cwd(), "webpack.config.js");
+		const {configFile} = opts
```

Comment:
Add spaces and `;`
`const { configFile } = opts;`

Reply:
done 👍